<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>935</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SWIR系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,5,1" columnstretch="1,1" rowminimumheight="0,0,0" columnminimumwidth="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_rgbCamera">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="title">
         <string>RGB Camera</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,1">
         <property name="topMargin">
          <number>9</number>
         </property>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBox_preview">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Preview</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_conn1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_disconn1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_cap1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Capture</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_blank">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_swirCamera">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="title">
         <string>SWIR Camera</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6" rowstretch="1,1">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox_device">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Device</string>
             </property>
             <widget class="QRadioButton" name="radioButton_FW">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>30</y>
                <width>101</width>
                <height>23</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>FilterWheel</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_LCTF">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>61</width>
                <height>23</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>LCTF          </string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_channel">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Channel</string>
             </property>
             <widget class="QComboBox" name="comboBox_wavelength">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>141</width>
                <height>20</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
             <widget class="QLabel" name="label_nm">
              <property name="geometry">
               <rect>
                <x>160</x>
                <y>30</y>
                <width>31</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>nm</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="checkBox_preview2">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Preview</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_conn2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_disconn2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_cap2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Capture</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_swirVideo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="title">
         <string>SWIR Camera Video</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_swirCamera">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_rgbVideo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="title">
         <string>RGB Camera Video</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_rgbCamera">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3,3">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="leftMargin">
         <number>200</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_startacq">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Acquisition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_stopacq">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Stop Acquisiton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_quit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_collect">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Collect Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_save">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Save Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_total">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Total Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1339</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_initial"/>
    <addaction name="action_config"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_initial">
   <property name="text">
    <string>Initialization</string>
   </property>
  </action>
  <action name="action_config">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
